dm/templates/instance/tests/integration/instance_1_nic.yaml

```diff
@@ -17,9 +17,20 @@ resources:
       machineType: f1-micro
       diskType: pd-ssd
       canIpForward: true
-      network: test-network-${RAND}
-      subnetwork: regions/us-central1/subnetworks/test-subnet-${RAND}
+      networks:
```

Comment:
why we don't use "network" in this case?

Reply:
Andriy, this test is dedicated for the networks property.
We have a separate test that should also check the network property.